analytics/src/main/kotlin/uk/govuk/app/analytics/extension/StringExtension.kt

```diff
@@ -1,19 +1,17 @@
 package uk.govuk.app.analytics.extension
 
 fun String.redactPii(): String {
-    val redactionText = "[REDACTED]"
-
     // Postcode
     val postcode = "([A-Za-z]{1,2}[0-9]{1,2}[A-Za-z]? *[0-9][A-Za-z]{2})"
-    var output = Regex(postcode).replace(this, redactionText)
+    var output = Regex(postcode).replace(this, "[postcode]")
 
     // Email
     val email = "[A-Za-z0-9+_.-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,}"
-    output = Regex(email).replace(output, redactionText)
+    output = Regex(email).replace(output, "[email]")
 
     // NI number
     val niNumber = "[A-Za-z]{2} *[0-9]{2} *[0-9]{2} *[0-9]{2} *[A-Za-z]"
-    output = Regex(niNumber).replace(output, redactionText)
+    output = Regex(niNumber).replace(output, "[NI number]")
```

Comment:
I wonder if we should add the word redacted back in here, like `[postcode - redacted]` for example? I'm thinking that just `[postcode]` might not be fully clear that's it's been redacted on purpose and could be seen as bug/issue.

Reply:
I matched the functionality in: https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/javascripts/govuk_publishing_components/analytics-ga4/pii-remover.js

but I'll ask Andrew which he would prefer

Reply:
I guessed you'd followed the web version, and given the context - just the type of thing being redacted is prob fine. But thought I'd check just in case

Reply:
Andrew said he'd prefer to keep it as is